src/main_thread/decrypt/find_key_system.ts

```diff
@@ -397,3 +396,35 @@ export default function getMediaKeySystemAccess(
     }
   }
 }
+/**
+ * Test a key system configuration, resolves with the MediaKeySystemAccess
+ * or reject if the key system is unsupported.
+ * @param {string} keyType - The KeySystem string to test (ex: com.microsoft.playready.recommendation)
+ * @param {Array.<MediaKeySystemMediaCapability>} keySystemConfigurations - Configurations for this keySystem
+ * @returns Promise resolving with the MediaKeySystemAccess. Rejects if unsupported.
+ */
+export async function testOneKeySystem(
```

Comment:
`One` is unnecessary in that function's name no?

Reply:
Yes you are right 👍